image_tools/src/showimage.cpp

```diff
@@ -82,6 +77,11 @@ class ShowImage : public rclcpp::Node
 
     RCLCPP_INFO(this->get_logger(), "Subscribing to topic '%s'", topic_.c_str());
     sub_ = create_subscription<sensor_msgs::msg::Image>(topic_, qos, callback);
+
+    if(window_name_ == ""){
```

Comment:
There's an uncrustify error here:
```suggestion
    if (window_name_ == "") {
```